src/main/java/org/opentripplanner/graph_builder/module/osm/OsmModule.java

```diff
@@ -541,14 +547,20 @@ private StreetEdge getEdgeForStreet(
       seb.withBogusName(true);
     }
 
-    // < 0.04: account for
-    if (carSpeed < 0.04) {
-      issueStore.add(new StreetCarSpeedZero(way));
-    }
-
```

Comment:
There was validation both here and in the WayPropertySet. I moved the issuestore entry to be added in the WayPropertySet (and also add issues when the speed limit is over the upper limit). I've slightly adjusted the minimum speed limit so it's 5km/h. I can make it lower if someone knows if there are cases where it can be less.